polkadot/runtime/parachains/src/assigner_coretime/tests.rs

```diff
@@ -291,6 +291,160 @@ fn assign_core_enforces_higher_or_equal_block_number() {
 	});
 }
 
+#[test]
+fn assign_core_inserts_into_empty_queue() {
+    let core_idx = CoreIndex(0);
+
+    new_test_ext(GenesisConfigBuilder::default().build()).execute_with(|| {
+        run_to_block(1, |_| None);
```

Comment:
```suggestion
        run_to_block(1, |_| None);
        let start_block = 10u32;
```

You can then use this instead of everywhere you have `BlockNumberFor::<Test>::from(10u32)`